lib/administrate/namespace/resource.rb

```diff
@@ -4,7 +4,7 @@ class Resource
       attr_reader :namespace, :resource
 
       def initialize(namespace, resource)
-        @namespace = namespace
+        @namespace = namespace.to_sym
```

Comment:
**question:** is this necessary? 

I have the feeling that `Resource` as an inner class of `Namespace` is only ever initialized from there. At least I found now other places when grepping the codebase. And in `Namespace`,  `namespace` is already made sure to be a symbol.

Reply:
Good catch! I removed this in the branch I'm trying to test and I think you're right.